### ISSUE TITLE
Make Caldav workflow more obvious

### ISSUE THREAD

**Issue by diegogangl** [getting-things-gnome/gtg#747]
This could involve redesigning the entire workflow or adding some text to the panel.
Also, documentation

**Comment by diegogangl** [comment]
Some notes of the top of my head (and Matrix):

- Tasks need to be tagged with 'DAV_[CALENDAR_NAME]` to be synced

> tasks needs to have a certain tag to select the collections they are gonna be synced on, and by default the plugin list all the collection and sync them all 
> I think it's doable to add an optionnal whitelist to select the collection we actually wanto see, and some documentation about the tag behavior 

>  in my implementation, gtg list available todo collection from the server, if the server lists all the collection then all the collection are accessible
> but yeah it should be whitelistable / configurable

(from @jaesivsm in Matrix)

**Comment by nekohayo** [comment]
I suppose this has been mostly addressed by @vansia43's neat additions to the user manual, or is there something else that remains to be done here?

**Comment by Neui** [comment]
From what I can see the main thing was about that it would sync from all collections the server reports rather than just a few that you could select. Some Interfaces also provide a link for CalDAV to a specific collection yet GTG syncs everything rather than looking at the "link-provided" collection.

**Comment by dmp1ce** [comment]
That's right. I was surprised all of my lists were synced from Nextcloud to GTG when I only expected one empty list where I could sync my GTG tasks.

**Comment by nekohayo** [comment]
Right, though that's tracked in issue #751, unless we would want to document this behavior in the user manual?

**Comment by diegogangl** [comment]
Eh, could have worded this better. This was mostly about the issue of having to tag tasks a certain way. I think we could find a more obvious way to link tags to calendars.

**Comment by vansia43** [comment]
> Right, though that's tracked in issue #751, unless we would want to document this behavior in the user manual?

Got a note in the caldav page about this: “The CalDAV provider might give you a URL for a specific calendar, but actually send the todos from all calendars. In this case, the CalDAV backend will end up syncing all calendars.”

So should be good there 😀
